test.js

```diff
@@ -59,3 +60,41 @@ describe('generateFileList', () => {
     expect(() => generateFileList([])).toThrow('Unexpected empty file list');
   });
 });
+
+describe('Running from cli', () => {
+  test('without unused translations', async () => {
+    let { stdout } = await execa('node', ['../../bin/cli'], {
+      cwd: `${__dirname}/fixtures/no-issues`,
+    });
+
+    expect(stdout).toMatch('No unused translations');
+  });
+
+  test('with unused translations', async () => {
+    expect(
+      execa('node', ['../../bin/cli'], { cwd: `${__dirname}/fixtures/unused-translations` })
+    ).rejects.toThrowError('Found 2 unused translations');
+  });
+
+  test('with missing translations', async () => {
+    expect(
+      execa('node', ['../../bin/cli'], { cwd: `${__dirname}/fixtures/missing-translations` })
+    ).rejects.toThrowError('Found 2 missing translations');
+  });
+
+  describe('with auto-fix', () => {
+    afterEach(async function () {
+      await execa('git', ['checkout', 'HEAD', 'fixtures/remove-unused-translations/translations'], {
+        cwd: __dirname,
+      });
```

Comment:
The auto-fix option actually changes the files, so this makes sure those changes are undone. It does feel a bit specific though, so any other ideas are welcome :)